packages/suite/src/views/wallet/staking/components/CardanoStake.tsx

```diff
@@ -137,16 +122,25 @@ export const CardanoStake = ({ account }: CardanoStakeProps) => {
                 </>
             )}
             <Actions>
-                {deviceAvailable.status && delegatingAvailable.status ? (
-                    actionButton
-                ) : (
-                    <Tooltip
-                        maxWidth={285}
-                        content={reasonMessageId ? <Translation id={reasonMessageId} /> : undefined}
-                    >
-                        {actionButton}
-                    </Tooltip>
-                )}
+                <DeviceButton
+                    isDisabled={
```

Comment:
nit: put it into constant to not bloat JSX

Reply:
845b57e974f906a1e9bd57de7d93c991fc58a27e